fetch.bs

```diff
@@ -1465,32 +1465,19 @@ null), run these steps. <var>processBody</var> must be an algorithm accepting a
  <li><p>If <var>taskDestination</var> is null, then set <var>taskDestination</var> to the result of
  <a>starting a new parallel queue</a>.
 
- <li><p>Let <var>promise</var> be the result of <a>fully reading body as promise</a> given
- <var>body</var>.
-
  <li><p>Let <var>fulfilledSteps</var> given a <a>byte sequence</a> <var>bytes</var> be to
```

Comment:
Should be renamed to successSteps/errorSteps since there is no more promise fulfillment/rejection going on.

Reply:
Done